### ISSUE TITLE
Best practice WCAG 2.1 AA - Document should have one main landmark

### ISSUE THREAD

**Issue by christinach** [pulibrary/Rails-project-Liz#16]
Resources:
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11.html
https://dequeuniversity.com/rules/axe/4.7/region?application=AxeChrome
